pom.xml

```diff
@@ -18,10 +18,9 @@
 		<module>spring-cloud-starter-dataflow-server-kubernetes</module>
 	</modules>
 
-
 	<properties>
 		<spring-cloud-dataflow.version>1.0.0.RELEASE</spring-cloud-dataflow.version>

```

Comment:
I think we can leave this one as is until we need some features/fixes from 1.0.1